.docker/entrypoint.sh

```diff
@@ -6,19 +6,22 @@ echo ""
 ls
 echo ""
 
-# run jekyll serve in hot-reload mode.
-# rerun whenever _config.yaml changes (jekyll hot-reload doesn't work with this file).
+# run cite process
+python3 _cite/cite.py
+
+# run jekyll serve in hot-reload mode
+# rerun whenever _config.yaml changes (jekyll hot-reload doesn't work with this file)
 watchmedo auto-restart \
     --debug-force-polling \
     --patterns="_config.yaml" \
     --signal SIGTERM \
     -- bundle exec jekyll serve --open-url --force_polling --livereload --trace --host=0.0.0.0 \
-    | sed 's/0.0.0.0/localhost/g' &
+    | sed 's/LiveReload address.*//g' & sed 's/0.0.0.0/localhost/g' &
```

Comment:
This looks a little odd; it looks like you pipe the output of `watchmedo` to `sed`, but then you start a new instance of `sed` immediately after that receives no input and is also backgrounded.

A quick fix would be to change the first `&` to a pipe, i.e. `|`. In that case you'd have two instances of `sed`, one which I assume removes the line starting with `LiveReload address` and the other which maps `0.0.0.0` to `localhost`. Alternatively, you could combine both those operations into a single `sed` invocation (my preference, but requires a little more editing), e.g. `sed 's/LiveReload address.*//g;s/0.0.0.0/localhost/g'`, note the `;` that divides two `sed` ops.

Reply:
It's strange that it worked at all if my syntax was wrong. I think it was working. Regardless, I like the single command better. I didn't know you could list multiple replacements.